orc8r/cloud/docker/controller/apidocs/v1/indexTemplated.html

```diff
@@ -0,0 +1,102 @@
+<!-- HTML for static distribution bundle build -->
+<!-- This file is pulled from swagger-ui repository -->
+<!-- If you bump swagger-ui version, make sure to update this file as well! -->
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <title>Magma API Docs</title>
```

Comment:
Don't worry about this extra html, I added it temporarily as a means of making sure that I dont make silly mistakes.